src/Controls/src/Core/HandlerImpl/ControlsWindow.cs

```diff
@@ -0,0 +1,35 @@
+using Microsoft.Maui;
+
+namespace Microsoft.Maui.Controls
+{
+	public class Window : VisualElement, IWindow
+	{
+		private Page _page;
+
+		public Window(IPage page)
+		{
+			Page = (Page)page;
+		}
+
+		public Page Page
```

Comment:
The Window children, should be an IPage or IView (including a Page, but also directly a Grid for example)

Reply:
I think on the Controls side though we are still forcing it to be a Page for now? Since we've always forced a page
Maybe later this year we can start allowing Views though. That would be neat!!